app/controllers/survey_controller.rb

```diff
@@ -1,4 +1,10 @@
 class SurveyController < ApplicationController
   def index
+    @questionList = Question.order(:order)
+    #@questions = [{:title => "How large is your hometown?", :answers => ["big", "small", "medium"]}, {:title => "How large is your hometown?", :answers => ["big", "small", "medium"]}]
```

Comment:
Line 3 just fetches all the questions from the database. Line 4 was me creating some "questions" just to have something to show on the survey page